src/Attribute/AsCommandArgument.php

```diff
@@ -5,6 +5,6 @@
 abstract class AsCommandArgument
 {
     public function __construct(
-        public readonly string|null $name = null,
+        public readonly null|string $name = null,
```

Comment:
I there a way to avoid this diff?
I really prefer string|null over null|string.
Also I prefer string? over string|null